### ISSUE TITLE
Artifactory and internal POM

### ISSUE THREAD

**Issue by CSBaum** [nebula-plugins/gradle-lint-plugin#30]
I am not really sure this is an 'issue', but i find that when i upload the gradle-lint jar into my Artifactory instance, it always pre-populates it with commons-lang:commons-lang:2.6, so i am always having to manually enter the group and artifact ids.  It seems to work fine when i import the actual gradle-lint pom.  

I am no artifactory expert so i can very well be doing something wrong ....

Sean

**Comment by jkschneider** [comment]
@CSBaum Curious... the lint plugin's pom looks good to me out in Maven Central. What mechanism are you using to upload it?

**Comment by CSBaum** [comment]
The upload option within the Artifactory web UI ...

**Comment by jkschneider** [comment]
Just tried that UI for the first time, and here is what I notice:

- When you are in "Single" mode, if you try to upload the jar it tries to select commons-lang as you say. I don't know it determines this from the jar, perhaps by looking at class signatures and trying to guess against known GAVs? Regardless, the guess is wrong as you point out.

- When you are in "Single" mode, if you try to upload the pom, it selects the correct GAV, but this is not super useful anyway.

- When you are in "Multi" mode, and you select both the JAR and pom (at the least), Artifactory makes no attempt to guess the GAV and instead requires you to type in the path manually.

Basically, I think this is just a not-so-useful feature of Artifactory.
